### PR TITLE
Hungarian translation update part 04

### PR DESCRIPTION
**Pull request by big4billy**
More translations and corrections till "MIDI"

### CODE REVIEW

**Comment by big4billy** [comment]
Dear Admins, @ihhub @zenseii 

We plan to complete the whole Hungarian translation till milestone 1.0.10. I already have all the translations offline, so I just need to put (copy&paste) in pull requests in 400 peaces parts. Is it possible? May you help us with quick reviews?

Kind regards!

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @big4billy , I left one comment here. Could you please take a look?

File: files/lang/hu.po [hunk 1]
```diff
@@ -2057,49 +2031,50 @@ msgid "Cannot recruit - you have too many Heroes."
...
 msgstr "Erődítmények"
 
 msgid "Farm"
-msgstr "Tanya"
+msgstr "Szántóföldek"
 
 msgid "Thatched Hut"
 msgstr "Nádkunyhó"
 
+
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Sure. Removed

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@big4billy , thank you very much for this update!

**Pull request merged**